### PR TITLE
Extend `context` models to cover its old `golang.org/x/net/context` home.

### PR DESCRIPTION
**Pull request by smowton**

### CODE REVIEW

File: ql/test/library-tests/semmle/go/frameworks/StdlibTaintFlow/Context.go [hunk 1]
```diff
@@ -4,6 +4,7 @@ package main
 
 import (
 	"context"
+	oldcontext "golang.org/x/net/context"
```
Comment (owen-mc) [inline ql/test/library-tests/semmle/go/frameworks/StdlibTaintFlow/Context.go:7]: Is the comment on line 1 still correct?

File: ql/src/semmle/go/frameworks/stdlib/Context.qll [hunk 2]
```diff
@@ -6,25 +6,33 @@ import go
 
 /** Provides models of commonly used functions in the `context` package. */
 module Context {
+  /**
+   * Gets the package name `context` or `golang.org/x/net/context`.
+   *
+   * The two packages are identical; before Go 1.7 it was only available in the
+   * under `golang.org/x`; as of Go 1.7 it is included in the standard library.
```
Comment (intrigus-lgtm) [inline ql/src/semmle/go/frameworks/stdlib/Context.qll:13]: "it was only available **in the under** `gol...`"
This does not sound right to me.

File: ql/test/library-tests/semmle/go/frameworks/StdlibTaintFlow/Context.go (hunk 1 above)
Comment (smowton) [reply ql/test/library-tests/semmle/go/frameworks/StdlibTaintFlow/Context.go:7]: Not quite... but close enough :) I hand-mimicked its style for the time being.

File: ql/src/semmle/go/frameworks/stdlib/Context.qll (hunk 2 above)
Comment (smowton) [reply ql/src/semmle/go/frameworks/stdlib/Context.qll]: Oops, fixed

**Review by owen-mc — APPROVED** [review]

**Pull request merged**